demo-shell/src/app.config.json

```diff
@@ -165,7 +165,10 @@
             }
           }
         }
-      ]
+      ],
+      "datePicker": {
+        "dateFormat": "DD-MMM-YY"
```

Comment:
This needs to be part of the "date range" widget settings block.

Reply:
I've made the requested change & made another commit to fix the "Type 'moment.Moment' is not assignable to type 'moment.Moment'" problem - this appeared because of having 2 different versions of Moment at the same time.  Denys, please review again, thank you.